bin/eds-apply-theme.js

```diff
@@ -85,8 +64,6 @@
   });
 
   try {
-    // Keys in the theme file must be a strict subset of those in the base file
-    isStrictSubset(baseTheme, localTheme);
```

Comment:
Themes can now introduce tokens that are local to the application, and aren't just a subset of what EDS provides. useful for adding custom colors, then mapping them to tier-2 tokens